src/query/join.rs

```diff
@@ -76,6 +76,7 @@ where
 mod tests {
     use crate::tests_cfg::{cake, cake_filling, cake_filling_price, filling, fruit};
     use crate::{ColumnTrait, DbBackend, EntityTrait, ModelTrait, QueryFilter, QueryTrait};
+    use pretty_assertions::assert_eq;
 
     #[test]
     fn join_1() {
```

Comment:
Can we have a slightly more elaborate unit test after here, even if the relation does not make sense?

Reply:
More level of joins?

Reply:
Yes, 3 or 4 levels maybe?

Reply:
Checkout a4f2e3c